docs-v2/content/en/docs/references/api-v2/grpc.md

```diff
@@ -7,7 +7,7 @@ weight: 30
 ******
 WARNING!!!
 
-The file docs/content/en/docs/references/api/grpc.md is generated based on proto/markdown.tmpl,
+The file docs-v1/content/en/docs/references/api/grpc.md is generated based on proto/markdown.tmpl,
```

Comment:
the related change under docs-v2 dir should be v2 right?

Reply:
Thanks, fixed